views/refinements/ingestion_metrics.view.lkml

```diff
@@ -14,26 +14,30 @@ view: +ingestion_metrics {
       year
     ]
     datatype: epoch
-    sql: UNIX_SECONDS(TIMESTAMP ${start_time}) ;;
+    sql: UNIX_SECONDS(${TABLE}.start_time) ;;
   }
 
   measure: total_entry_number {
```

Comment:
Check with Ryan if we need to do this for Looker Block. This caused multiple issues with embedded looker.